lib/vue-scroll.js

```diff
@@ -60,15 +66,39 @@
                   
                 if(typeof Q._initialized == 'undefined'){
                     
-                  Q.prototype.bind = (function(element, eventType, func){
-                    var funcs;
-                    
-                    if(elements.indexOf(element) < 0){
+                  Q.prototype.bind = (function(element, eventType, func, arg){
+                    var funcs, i, length = elements.length, throttle;
+
+                    if(arg){
+                        var lastCallTimestamp, timeout = null, value, pos, caller = function(){
```

Comment:
You mean the **scrollData** data?  Yes, you are right, please go ahead. 

And I am going to refactor the part that you have replaced the **indexOf** using for loop. It's a little bit hard to read currently(The **i** and the **length**), I think.  A compatible **indexOf** may be better. And we need more comments for the arg. And more description about it on README as a new important section.